draft-packages/illustration/KaizenDraft/Illustration/Scene.tsx

```diff
@@ -88,20 +91,23 @@ export const BrandMomentError = ({
 
 export const BrandMomentNewAccountOnboarding = (props: SceneProps) => (
   <Base
+    aspectRatio={props.enableAspectRatio ? "landscape" : undefined}
```

Comment:
As someone who loves their tests, I'd would want one here as well, but that'll get super tedious quick, as we have so many Illustration components.
Though it could just be one simple test asserting `aspectRatio` is not `undefined` when `enableAspectRatio` is `true` and then just map that one test over all illustration components.